HW6/HW6.py

```diff
@@ -1,19 +1,542 @@
+# import os
+import sys
+
+import charts
+import correlation as dafrenchy_correlation
 import pandas as pd
 import sqlalchemy
 
-# Thanks dafrenchy for this plug-and-play mariadb connection.
-db_user = "root"
-db_pass = "secret"  # pragma: allowlist secret
-db_host = "localhost:3306"
-db_database = "mysql"
-connect_string = (
-    f"mariadb+mariadbconnector://{db_user}:{db_pass}@{db_host}/{db_database}"
-)
-# pragma: allowlist secret
-
-sql_engine = sqlalchemy.create_engine(connect_string)
-
-query = """
-    SELECT * FROM DB
-"""
-df = pd.read_sql_query(query, sql_engine)
+# import statsmodels.api as sm
+from plotly import figure_factory as ff
+from sklearn.ensemble import AdaBoostClassifier, RandomForestClassifier
+
+# from sklearn.linear_model import LinearRegression, LogisticRegression
+from sklearn.metrics import confusion_matrix
+
+# from sklearn.tree import DecisionTreeClassifier
+
+
+def main():
+    # prompt = (
+    #     "please enter the location of the "
+    #     " HW6 dir on the host machine ending in "
+    #     "HW6"
+    # )
+    # cwd = '/Users/salt/PycharmProjects/BDA3/src/BDA2/HW6'
+    # path = "file://" + cwd + '/'
+    # testing only
+    path = ""
+    # print("Here's the working directory where you can find the files:")
+    # print(os.getcwd())
+
+    # Thanks dafrenchy for this plug-and-play mariadb connection.
+    db_user = "root"
+    db_pass = "secret"  # pragma: allowlist secret
+    db_host = "db-service:3306"
+    db_database = "baseball"
+    connect_string = (
+        f"mariadb+mariadbconnector://{db_user}:{db_pass}@{db_host}/{db_database}"
+    )
+    # pragma: allowlist secret
+
+    sql_engine = sqlalchemy.create_engine(connect_string)
+
+    query = """
+        SELECT * FROM home_team_wins
+    """
+    df = pd.read_sql_query(query, sql_engine)
+    df.dropna(inplace=True)
+    # print(df.head())
+
+    # classify predictors here manually
+    response = "HOME_TEAM_WINS"
+    response_type = "categorical"
+
+    predictors = [
```

Comment:
I like how you are specific with what your predictors are and splitting them by cat/cont